src/index.html

```diff
@@ -42,45 +42,45 @@ <h3>
           <p>To enter your information into the FitLit activity tracker please select the categories button above</p>
         </article>
       <article class="addActivity__article">
-        <form method="POST">
+        <form>
           <label for="form__control--date1">Date
-          <input aria-required="true" id="form__control--date1" type="text" name="date" placeholder="YYYY/DD/MM">
+          <input class="form-control" aria-required="true" id="form__control--date1" type="text" name="date" placeholder="YYYY/DD/MM">
           </label>
           <label for="form__control--steps">Number of Steps
-          <input aria-required="true" id="form__control--steps" type="number" name="number" pattern="/^[0-9]+$/" min="1" max="20000" placeholder="#">
+          <input class="form-control" aria-required="true" id="form__control--steps" type="number" name="number" pattern="/^[0-9]+$/" min="1" max="20000" placeholder="#">
           </label>
           <label for="form__control--minutes">Minutes Active
-          <input aria-required="true" id="form__control--minutes" type="number" name="number" pattern="/^[0-9]+$/" min="1.0" max="400" placeholder="#">
+          <input class="form-control" aria-required="true" id="form__control--minutes" type="number" name="number" pattern="/^[0-9]+$/" min="1.0" max="400" placeholder="#">
           </label>
           <label for="form__control--stairs">Flights of Stairs
-          <input aria-required="true" id="form__control--stairs" type="number" name="number" pattern="/^[0-9]+$/" min="1.0" max="500" placeholder="#">
+          <input class="form-control" aria-required="true" id="form__control--stairs" type="number" name="number" pattern="/^[0-9]+$/" min="1.0" max="500" placeholder="#">
```

Comment:
The pattern you put in for all the forms is great, Cameron!  When I pulled this down and tested it, I couldn't even enter a letter.